src/system/Notice/Notice.tsx

```diff
@@ -5,43 +5,61 @@
  */
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import { MdError, MdWarning, MdCheckCircle, MdInfo } from 'react-icons/md';
 
 /**
  * Internal dependencies
  */
 import { Box, Flex, Heading, Card } from '../';
+import { ThemeUIStyleObject } from 'theme-ui';
 
-const colorSystemVariant = color =>
-	( { warning: 'warning', error: 'error', alert: 'error', success: 'success', info: 'info' }[
-		color
-	] );
+interface NoticeIconProps {
+	color: string;
+	variant: keyof COLOR_VARIANTS;
+}
 
-const NoticeIcon = ( { color, variant } ) => {
-	let Icon = MdWarning;
+export interface NoticeProps {
+	children: React.ReactNode;
+	inline?: boolean;
+	sx?: ThemeUIStyleObject;
+	title?: React.ReactNode;
+	variant?: keyof COLOR_VARIANTS;
+	headingVariant?: React.ElementType;
+	className?: string;
+}
+type COLOR_VARIANTS = {
+	warning: string;
+	error: string;
+	alert: string;
+	success: string;
+	info: string;
+};
+const COLOR_VARIANTS_MAPPING: COLOR_VARIANTS = {
+	warning: 'warning',
+	error: 'error',
+	alert: 'error',
+	success: 'success',
+	info: 'info',
+};
+const colorSystemVariant = ( color: keyof COLOR_VARIANTS ) => COLOR_VARIANTS_MAPPING[ color ];
```

Comment:
```suggestion
```

Reply:
then remove `const systemVariant = colorSystemVariant( variant );` below and `systemVariant` => `variant`

Reply:
@chriszarate I think the main issue here is the `alert` that gets converted to `error`. In your example it'll mean we'll get an `alert` variant, but we want the error one. I agree though that this is not the cleanest approach. Let me think if we can implement it differently.

Reply:
Oh right, I think we should just unwind it. Either add an `alert` variant that duplicates the styling of `error`, or update dashboard to pass `error`

Reply:
Ok, I went ahead and extended the theme so we have the `alert` color support. 
Done in https://github.com/Automattic/vip-design-system/pull/253/commits/847f53317407c15f3976524b29f32720dd03fa12